libs/fileUtils.js

```diff
@@ -0,0 +1,24 @@
+const fs = require('fs');
+
+const cli = require('child_process').execSync;
+
+const rm = (filename) => cli(`rm -rf ${filename}`);
+
+const read = (filename) => fs.readFileSync(filename, 'utf8');
+
+const mkdir = (filename) => cli(`mkdir -p ${filename}`);
```

Comment:
Do we need to do anything to make this relative to the build directory, or is the `cli` interface already do that?

Reply:
long answer above, short answer no